frontend/src/components/pages/TeamDashboard/Presentation.js

```diff
@@ -33,11 +33,18 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const ReviewLineChart = ({ data, minimum, maximum }) => {
-  // TODO calculate maximum and minimum so that all the graphs have the same proportions
-
+const ReviewLineChart = ({ data, YAxisData }) => {
+  let reviewsArr = []
+  for(let i in YAxisData){
```

Comment:
can you also use the appropriate naming convention, camelCase for YAxisData, YAxisArr